File: Eclipse 2/codeLM starter-code/Python/new_wave_slots.py
def new_wave_slots(given_slot):
  # Your code here


if __name__ == '__main__':
  given_slot_row = int(input())
  given_slot_col = int(input())
  given_slot = []
  for _ in range(given_slot_row):
    given_slot.append([])
    for _ in range(given_slot_col):
      given_slot[-1].append(input())

  function_call_result = new_wave_slots(given_slot)
  for x in function_call_result:
    for y in x:
      print(y, end=' ')
    print()

File: Eclipse 2/codeLM starter-code/Java/NewWaveSlots.java
import java.util.Scanner;

class NewWaveSlots {
    static char[][] newWaveSlots(char[][] givenSlot) {
        // Your code here
    }

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int givenSlotRow = Integer.parseInt(s.nextLine());
        int givenSlotCol = Integer.parseInt(s.nextLine());
        char[][] givenSlot = new char[givenSlotRow][givenSlotCol];
        for (int i = 0; i < givenSlotRow; i++) {
            for (int j = 0; j < givenSlotCol; j++) {
                givenSlot[i][j] = s.nextLine().charAt(0);
            }
        }

        char[][] functionCallResult = newWaveSlots(givenSlot);
        for (int i = 0; i < functionCallResult.length; i++) {
            for (int j = 0; j < functionCallResult[i].length; j++) {
                System.out.print(functionCallResult[i][j]);
                System.out.print(" ");
            }
            System.out.println();
        }
    }
}